src/azul/project/hca/transformers.py

```diff
@@ -67,11 +66,27 @@ def _project_dict(project: api.Project) -> JSON:
     }
 
 
-def _specimen_dict(entity: api.LinkedEntity, visited_entities) -> JSON:
-    visitor = SpecimenVisitor(visited_entities)
-    entity.accept(visitor)
-    entity.ancestors(visitor)
-    return visitor.merged_specimen
+def _sample_dict(sample: api.Biomaterial) -> JSON:
+    return {
+        'sample_entity_type': api.schema_names[type(sample)],
```

Comment:
```suggestion
        'sample_type': api.schema_names[type(sample)],
```

And: I want the value to be consistent with our notion of entity type. So for `specimen_from_organism` this should be `specimens`. I know plural is odd but we'll switch from plural to singular at some point: https://github.com/DataBiosphere/azul/issues/824